### PR TITLE
Tank 232 user card search bar parity debug

### PR DESCRIPTION
**Pull request by wmitch34**

### CODE REVIEW

**Review by thomas-harmeyer — CHANGES_REQUESTED** [review]

File: frontend/src/components/forms/EditUser.tsx [hunk 1]
```diff
@@ -42,6 +30,10 @@ export default function EditUserModal({
     return null;
   }
 
+  useEffect(() => {
+    console.log('hit');
```
[the remark this answers is not in the document]
Comment (wmitch34) [reply frontend/src/components/forms/EditUser.tsx]: done

File: frontend/src/components/forms/CreateUser.tsx [hunk 2]
```diff
@@ -13,20 +13,21 @@ import {
...
 
-export const userSchema = z.object({
-  id: z.number().int(),
-  firstName: z.string().optional(),
-  middleName: z.string().optional(),
-  lastName: z.string().optional(),
-  address: z.string().optional(),
-  phone: z.string().optional(),
+// export const userSchema = z.object({
+//   id: z.number().int(),
```
[the remark this answers is not in the document]
Comment (wmitch34) [reply frontend/src/components/forms/CreateUser.tsx:19]: done

File: frontend/src/components/UserSearchBar.tsx [hunk 3]
```diff
@@ -48,7 +48,7 @@ export default function UserSearchBar({
       options={optionsList
         .slice()
         .sort((a, b) => a.firstName.localeCompare(b.firstName))}
-      groupBy={(option) => option.firstName.charAt(0).toUpperCase()}
+      groupBy={(option) => option.firstName?.charAt(0).toUpperCase() ?? ''}
```
[the remark this answers is not in the document]
Comment (wmitch34) [reply frontend/src/components/UserSearchBar.tsx:57]: done

**Review by wmitch34 — COMMENTED** [review]
made the requested changes

**Pull request closed without merging**